### ISSUE TITLE
NPG data in Browse App: "Secondary Artist/Maker" not included

### ISSUE THREAD

**Issue by si-npg** [american-art/npg#77]
I'm pretty sure this role was not included for the browse app, and it's fairly important for NPG objects. I can elaborate and give examples of where it matters if need be.

**Comment by workergnome** [comment]
I agree.   Roles for multiple creators are a major absence in the current model and mappings.

**Comment by si-npg** [comment]
High priority
